appendices/F/5_fix_too_big/palindrome/palindrome.go

```diff
@@ -0,0 +1,38 @@
+package palindrome
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+)
+
+// IsPalindromeNumber returns if an integer is readable in both ways.
+// Only unsigned int are supported.
+// "1221" is a palindrome
+// "01" is not a palindrome
+// "-1" is not a palindrome
+func IsPalindromeNumber(s string) (bool, error) {
+	// value could be bigger than max int e.g.: 10000000000000000000
+	if len(s) > 10 {
```

Comment:
(we should still hit some errors here - `2^32+1` is ~ `4 * 10 ^9`. But it's a safe test.